go/worker/common/committee/node.go

```diff
@@ -62,13 +66,54 @@ var (
 		},
 		[]string{"runtime"},
 	)
+	workerIsExecutorWorker = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_is_executor_worker",
+			Help: "1 if worker is currently an executor worker, 0 otherwise.",
+		},
+		[]string{"runtime"},
+	)
+	workerIsExecutorBackup = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_is_executor_backup_worker",
+			Help: "1 if worker is currently an executor backup worker, 0 otherwise.",
+		},
+		[]string{"runtime"},
+	)
+	committeePeers = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_committee_peers",
+			Help: "Number of committee peers.",
+		},
+		[]string{"runtime"},
+	)
+	livenessTotalRounds = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "oasis_worker_liveness_total_rounds",
+			Help: "Number of total rounds in last epoch.",
+		},
+		[]string{"runtime"},
+	)
+	livenessLiveRounds = prometheus.NewGaugeVec(
```

Comment:
Could we also add a percentage (live / total). Maybe it should report 100% if the node is not in the committee just to make alerting easier?

Reply:
Added